### PR TITLE
Implement UI for welcome screen

### PR DESCRIPTION
**Pull request by julianhahn**
Create a welcome screen following the design from figma.
2020-12-29 17_52_11-Backend list

### CODE REVIEW

**Review by JanEberlein — APPROVED** [review]

File: packages/app/package.json [hunk 1]
```diff
@@ -16,6 +16,8 @@
     "@feathersjs/socketio-client": "4.5.11",
     "@fortawesome/fontawesome-free": "^5.15.1",
     "@project-leek/commons": "*",
+    "typeface-baloo": "^1.1.13",
+    "typeface-roboto": "^1.1.13",
```
Comment (anbraten) [inline packages/app/package.json:20]: Please pin dependencies and run `yarn` again to update `yarn.lock`

```suggestion
    "typeface-baloo": "1.1.13",
    "typeface-roboto": "1.1.13",
```

File: packages/app/src/router/index.ts [hunk 2]
```diff
@@ -51,7 +54,7 @@ router.beforeEach(async (to, _, next) => {
   const pageAuthentication = to.meta.authentication || 'needed';
 
   if (pageAuthentication === 'needed' && !isAuthenticated.value) {
-    next({ name: 'login' });
+    next({ name: 'welcome' });
```
Comment (anbraten) [inline packages/app/src/router/index.ts:57]: Same as above.

File: packages/app/src/App.vue [hunk 3]
```diff
@@ -23,7 +23,7 @@ export default defineComponent({
         // if not authenticated and on page that requests authentication
         const pageAuthentication = route.meta.authentication || 'needed';
         if (pageAuthentication === 'needed' && !_isAuthenticated) {
-          router.replace({ name: 'login' });
+          router.replace({ name: 'welcome' });
```
Comment (anbraten) [inline packages/app/src/App.vue:26]: This now means: If you go to (or if you are still on) a page which requires authentication and you are not authenticated, redirect the user to welcome. 

As a user I would still expect to land on the login page in such a case. The welcome screen should come each time I open the app without requesting a specific page (like opening the first time with the base url).

**Review by anbraten — CHANGES_REQUESTED** [review]
Why have you added `tailwind.css` to our assets folder? IMO Tailwind should come from `node_modules`?

I like the cube image, but it would be nice if it could be quite a bit smaller (file-size) to improve loading speed.

**Comment by julianhahn** [comment]
> Why have you added `tailwind.css` to our assets folder? IMO Tailwind should come from `node_modules`?
> 
> I like the cube image, but it would be nice if it could be quite a bit smaller (file-size) to improve loading speed.

I'm not quite sure, if there is a command in our packages files, which build the tailwind.css file with the current tailwind.config file. Up until now, we alway build the file with it's npx command in the package json. To ensure, that every collaborator has the currently builded file, I added it. 

I'm open for other solutions.

File: packages/app/src/App.vue (hunk 3 above)
Comment (julianhahn) [reply packages/app/src/App.vue:26]: In our current design, there is no other page with the option to choose your login method. What should be displayed on an 'login' screen, which is not already on the welcome screen?

File: packages/app/package.json (hunk 1 above)
Comment (julianhahn) [reply packages/app/package.json]: fixed in f513c5e0436795129e3f0e0f75e7d59c6dc1f421

File: packages/app/src/App.vue (hunk 3 above)
Comment (anbraten) [reply packages/app/src/App.vue:26]: I agree with that. So can we delete the old login page and all links / references to it then?

Comment (julianhahn) [reply packages/app/src/App.vue:26]: I would suggest that.

**Comment by anbraten** [comment]
> > Why have you added `tailwind.css` to our assets folder? IMO Tailwind should come from `node_modules`?
> > I like the cube image, but it would be nice if it could be quite a bit smaller (file-size) to improve loading speed.
> 
> I'm not quite sure, if there is a command in our packages files, which build the tailwind.css file with the current tailwind.config file. Up until now, we alway build the file with it's npx command in the package json. To ensure, that every collaborator has the currently builded file, I added it.
> 
> I'm open for other solutions.

Ahh okay, I never tried that config part of tailwind. It's fine for me if you say that this is the best way to do it.

**Comment by JanEberlein** [comment]
> > > Why have you added `tailwind.css` to our assets folder? IMO Tailwind should come from `node_modules`?
> > > I like the cube image, but it would be nice if it could be quite a bit smaller (file-size) to improve loading speed.
> > 
> > 
> > I'm not quite sure, if there is a command in our packages files, which build the tailwind.css file with the current tailwind.config file. Up until now, we alway build the file with it's npx command in the package json. To ensure, that every collaborator has the currently builded file, I added it.
> > I'm open for other solutions.
> 
> Ahh okay, I never tried that config part of tailwind. It's fine for me if you say that this is the best way to do it.

The [documentation](https://tailwindcss.com/docs/installation#building-your-css) only says that the styles need to be compiled via postcss, but nothing about having the styles in our repo.
I've tried to use your `tailwind.config.js` without the `tailwind.css` and it worked fine without problems. That would mean that our build process already supported using `tailwind.config.js`. I think we could safely go back to the old version of `tailwind.css`, but I would feel much safer about that recommendation if somebody else tested this as well 😅

**Review by anbraten — APPROVED** [review]
Thanks for shrinking the cube image. LGTM

**Pull request merged**